src/Twig/Extension/FlashMessageRuntime.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Twig\Extension;
+
+use Sonata\CoreBundle\FlashMessage\FlashManager;
+
+/**
+ * This is the Sonata core flash message Twig runtime.
+ *
+ * @author Vincent Composieux <composieux@ekino.com>
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ */
+class FlashMessageRuntime
+{
+    /**
+     * @var FlashManager
+     */
+    protected $flashManager;
```

Comment:
Please use `final` and `private` as much as you can

Reply:
I think this would break the migration path as users currently extending StatusExtension/FlashMessageExtension could not extends anything anymore. But I can do it if you want.

Reply:
I think they can still rely on service decoration, right, so it would make things a bit harder but still possible.